apps/pie-storybook/stories/pie-link.stories.ts

```diff
@@ -82,19 +92,27 @@ const linkStoryMeta: LinkStoryMeta = {
             if: { arg: 'tag', eq: 'button' },
         },
         isBold: {
-            description: 'If `true`, makes the link text bold',
+            description: 'If `true`, makes the link text bold.',
             control: 'boolean',
             defaultValue: {
                 summary: false,
             },
         },
         isStandalone: {
-            description: 'If `true`, the link will be treated as a block box',
+            description: 'If `true`, the link will be treated as a block box.',
```

Comment:
what does this mean @raoufswe ?

Reply:
should it be 'block element'?

Reply:
yea that would work too, block-level boxes are block containers/elements.

Reply:
i will update it in my form-label PR 🥇